Add Nextjournal option, take two

mk opened the pull request:
Hi @domoritz! This adds the option to view & run these notebooks in Nextjournal. We made sure all notebooks are runnable as-is now. This supersedes #6.

domoritz commented on README.md:
```diff
@@ -77,6 +85,7 @@ The visualization curriculum can be used either online or on your local computer
 
 - To read JavaScript notebooks online using [Observable](https://observablehq.com/), click the "Observable" links above.
 - To read Python notebooks online using [Colab](https://colab.research.google.com/), click the "Colab" links above.
+- To read Python notebooks online using [Nextjournal](https://nextjournal.com/), click the "Nextjournal" links above.
```
Ahh, I forgot to mention Jupyter book. Can you add it here?

domoritz commented on README.md:
```diff
@@ -11,55 +11,62 @@ A data visualization curriculum of interactive notebooks, using [Vega-Lite](http
      [Open in Colab](https://colab.research.google.com/github/uwdata/visualization-curriculum/blob/master/altair_marks_encoding.ipynb) |
-     [Open in Observable](https://observablehq.com/@uwdata/data-types-graphical-marks-and-visual-encoding-channels)
+     [Open in Observable](https://observablehq.com/@uwdata/data-types-graphical-marks-and-visual-encoding-channels) |
+     [Open in Nextjournal](https://github.nextjournal.com/uwdata/visualization-curriculum/blob/master/altair_marks_encoding.ipynb)
```
Nextjournal should probably come before observable since observable is js and the rest are python. Can you make that a bit clearer (e.g. add **Python** or **JavaScript** to the description somewhere)?

mk commented:
@domoritz I tried addressing your comments in https://github.com/uwdata/visualization-curriculum/blob/11d4fb8ddaa66988f42b4bfedb080cfe74318b71/README.md. Let me know if you think this works or would like to see further changes.
